pkgs/development/python-modules/flask-httpauth/default.nix

```
@@ -1,16 +1,18 @@
-{ lib, python, isPy3k, buildPythonPackage, fetchPypi, flask }:
+{ lib, python, isPy3k, buildPythonPackage, fetchPypi, pytest, flask }:
 
 buildPythonPackage rec {
   pname = "Flask-HTTPAuth";
```

```suggestion
  pname = "flask-httpauth";
```

We should follow [PEP-503](https://nixos.org/manual/nixpkgs/stable/#contributing) here.

---

Thanks for your feedback. I changed the package name and the source's `pname` accordingly.